Tools/Kong/README.md

```diff
@@ -32,3 +32,7 @@ After the configurations for the environments have been generated, they can be p
 ```shell
 ./publish.sh
 ```
+
+## Publish to Kong using the pipeline
+
+The actual publishing to Cong dev, test and prod is done from a Azure DevOps pipeline using the preconfigured `cong - [env]` library variable groups.
```

Comment:
Nit - typo in the name Cong should be Kong

Reply:
fixed